Started GET "/babel/parser/lib/index.js" for 127.0.0.1 at 2022-04-18 17:02:30 -0400
  
ActiveRecord::ConnectionNotEstablished (connection to server on socket "/tmp/.s.PGSQL.5432" failed: fe_sendauth: no password supplied
):
  
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.5) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.5) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.5) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.5) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/host_authorization.rb:148:in `call'
railties (6.1.5) lib/rails/engine.rb:539:in `call'
puma (5.6.4) lib/puma/configuration.rb:252:in `call'
puma (5.6.4) lib/puma/request.rb:77:in `block in handle_request'
puma (5.6.4) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.6.4) lib/puma/request.rb:76:in `handle_request'
puma (5.6.4) lib/puma/server.rb:441:in `process_client'
puma (5.6.4) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/babel/parser/lib/index.js" for 127.0.0.1 at 2022-04-18 17:59:00 -0400
  
ActiveRecord::ConnectionNotEstablished (connection to server on socket "/tmp/.s.PGSQL.5432" failed: fe_sendauth: no password supplied
):
  
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.5) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.5) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.5) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.5) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/host_authorization.rb:148:in `call'
railties (6.1.5) lib/rails/engine.rb:539:in `call'
puma (5.6.4) lib/puma/configuration.rb:252:in `call'
puma (5.6.4) lib/puma/request.rb:77:in `block in handle_request'
puma (5.6.4) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.6.4) lib/puma/request.rb:76:in `handle_request'
puma (5.6.4) lib/puma/server.rb:441:in `process_client'
puma (5.6.4) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/babel/parser/lib/index.js" for 127.0.0.1 at 2022-04-18 17:59:00 -0400
  
ActiveRecord::ConnectionNotEstablished (connection to server on socket "/tmp/.s.PGSQL.5432" failed: fe_sendauth: no password supplied
):
  
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.5) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.5) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.5) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.5) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/host_authorization.rb:148:in `call'
railties (6.1.5) lib/rails/engine.rb:539:in `call'
puma (5.6.4) lib/puma/configuration.rb:252:in `call'
puma (5.6.4) lib/puma/request.rb:77:in `block in handle_request'
puma (5.6.4) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.6.4) lib/puma/request.rb:76:in `handle_request'
puma (5.6.4) lib/puma/server.rb:441:in `process_client'
puma (5.6.4) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/babel/parser/lib/index.js" for 127.0.0.1 at 2022-04-18 17:59:00 -0400
  
ActiveRecord::ConnectionNotEstablished (connection to server on socket "/tmp/.s.PGSQL.5432" failed: fe_sendauth: no password supplied
):
  
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.5) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.5) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.5) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.5) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.5) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.5) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.5) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.5) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.5) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.5) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.5) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.5) lib/action_dispatch/middleware/host_authorization.rb:148:in `call'
railties (6.1.5) lib/rails/engine.rb:539:in `call'
puma (5.6.4) lib/puma/configuration.rb:252:in `call'
puma (5.6.4) lib/puma/request.rb:77:in `block in handle_request'
puma (5.6.4) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.6.4) lib/puma/request.rb:76:in `handle_request'
puma (5.6.4) lib/puma/server.rb:441:in `process_client'
puma (5.6.4) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
